test/faye-redis.unit.js

```diff
@@ -0,0 +1,71 @@
+const { expect } = require('chai');
+const faye = require('faye');
+const hat = require('hat');
+const redis = require('redis');
+
+const fayeRedis = require('../faye-redis');
+const utils = require('./utils');
+
+describe('Redis engine', function() {
+    const namespace = 'faye';
+    let channel;
+    let clientId;
+    let engineOpts;
+    let redisClient;
+    let engine;
+    let bayeux;
+
+    before(() => {
+        redisClient = redis.createClient();
+    });
+
+    after(async () => {
+        await utils.invokeAsPromise(redisClient, 'quit');
+    });
+
+    beforeEach(() => {
+        bayeux = new faye.NodeAdapter({ mount: '/faye', timeout: 45 });
+        engine = fayeRedis.create(bayeux._server._engine, { namespace });
+
+        clientId = hat();
+        channel = hat();
+    });
+
+    afterEach(() => {
+        engine.disconnect();
+        bayeux.close();
+    });
+
+    describe('#subscribe', function() {
+        it('should subscribe the client to the channel', async () => {
+            engine.subscribe(clientId, channel);
+            await utils.delay(500);
+
+            const channelIsMember = await utils.invokeAsPromise(
+                redisClient,
+                'sismember',
+                `${namespace}/clients/${clientId}/channels`,
+                channel,
+            );
+            expect(channelIsMember).to.be.eql(1);
+
+            const clientIsMember = await utils.invokeAsPromise(
+                redisClient,
+                'sismember',
+                `${namespace}/channels${channel}`,
```

Comment:
```suggestion
                `${namespace}/channels/${channel}`,
```

Reply:
Not sure if that's wanted or if that's a typo, but there is no slash 
https://github.com/smooch/faye-redis-node/blob/master/faye-redis.js#L127 :)

Reply:
Oh I see 👍